app/models/spree/gateway/stripe_connect.rb

```diff
@@ -3,6 +3,8 @@ class Gateway
     class StripeConnect < Gateway
       preference :enterprise_id, :integer
 
+      validate :ensure_enterprise_selected
```

Comment:
Isn't this custom validation exactly what  `validates : preferred_enterprise_id, presence: { message: I18n.t(:error_required) }` does?

Reply:
@sauloperez Nope, it's using the preferences gem, which by some magic transforms a nil value into zero, and a value of zero passes the presence validation